src/components/Borrowpage.jsx

```diff
@@ -3,9 +3,58 @@ import moment from 'moment'
 import Calendar from 'components/Calendar.jsx'
 
 export default class Borrowpage extends React.Component {
+	
+	constructor(props){
+		super(props);
+		this.state = {
+			date: moment(),
+			selectedDays: []
+		};
+		this.toggle = this.toggle.bind(this);
+	}
+
+
+	toggle(e) {
```

Comment:
if isPicked then delete the date.
else 
    if length <= 6
        add the date
    else alert.